src/app/parcels/form/ParcelForm.tsx

```diff
@@ -183,14 +184,13 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
             return;
         }
 
-        const packingDate = dayjs(fields.packingDate).toISOString();
+        const packingDate = getDbDate(dayjs(fields.packingDate));
 
         let collectionDateTime = null;
         if (fields.shippingMethod === "Collection") {
-            collectionDateTime = mergeDateAndTime(
-                fields.collectionDate!,
-                fields.collectionTime!
-            ).toISOString();
+            collectionDateTime = getDbDate(
+                mergeDateAndTime(fields.collectionDate!, fields.collectionTime!)
+            );
```

Comment:
Why do we only output the date for collection date time?

Reply:
Sorry, this was a total mistake. Got over excited with the getDbDate function and I didn't consider the fact that the collection date time is stored in the database with the time as well as the date. We don't need this change